test/zmq_proxy.xpub-xsub.js

```diff
@@ -152,4 +160,8 @@ describe('proxy.xpub-xsub', function() {
       },200)
     }
   })
-});
\ No newline at end of file
+});
+
+function port(){
+  return Math.floor(Math.random()*10000)
```

Comment:
This could yield port 0. It's probably best to stay away from low port numbers, I reckon.